architecture/Threatmodel.md

```diff
@@ -0,0 +1,128 @@
+## 1. Contact informatie bepalen Risico’s 
+Hiervoor wordt verwezen naar het document Crypto Raamwerk in https://github.com/minvws/nl-covid19-notification-app-coordination/tree/master/architecture
+Een kleine opmerking valt wel te maken over de test optie van het GAEN protocol. Deze is noodzakelijk om een snelle feedback te krijgen tijdens de test van de programmatuur, maar is niet wenselijk tijdens productie. De vraag is hoe toegang tot een systeemtest in de App wordt geregeld.
+## 2. Informatie over besmette contacten verkrijgen Risco's 
+De app moet regelmatig informatie van een centrale server ophalen met daarin de Temporary Exposure keys van besmette personen. 
+### Spoofing: 
+Niet van toepassing. 
+### Tampering 
+Aangenomen wordt dat alleen een proces op de server(s) de nieuwe bestanden met daarin Temporary Exposure Keys kan generen. Gebruikers toegang tot deze bestanden behalve voor backup doeleinden dient te worden vermeden. 
+### Non Repudiation 
+Niet van Toepassing 
+###  Information Disclosure
+Hierbij kunnen andere clients dan de officiele app ook de Temporary exposure keys downloaden.  Om dit te vermijden dienen er maatregelen genomen worden zodat de informatie alleen beschikbaar is voor de app. 
+Ook dient de verbinding tussen de App en de centrale server beveiligd te zijn, zodat deze niet ontsleuteld kan worden door tussenliggende partijen. TLS met daarbij Certificate pinning wordt aangeraden. 
+De server dient ook beschermt te zijn tegen ongeauthorizeerde toegang via andere kanalen dan de download faciliteit die de app gebruikt. De gebruikelijke OS permissie, OS hardening, software update en authenticatie maatregelen dienen genomen te worden. 
+### Denial of Service
+Omdat elke App de bestanden download kan men eenvoudig de adressen van de servers achterhalen door de netwerk activiteit van de app te observeren. Dus het is een risico dat deze servers blootgesteld worden aan Denial of Service Attacks. 
+### Elevation of Privilege
+Niet van toepassing privileges/authenticatie worden in dit proces zoals nu bekend niet gebruikt.
+
+## 3. Het uitvoeren van een test en het doorgeven van een positieve test uitslag.
+### Spoofing
+De test en uitslagen worden verwerkt via een web applicatie. TLS encryptie dient te worden uitgevoerd. De OWASP richtlijnen voor veilige web applicaties, om risico's zoals sql injection, cross site scripting en dergelijke uit te sluiten.   
+### Tampering
+Toegang tot de databases waarin tests worden opgelagen dient goed te worden geregeld. Daarnaast dient het systeem van de TAN code's ook cryptografisch geanalyseerd te worden. 
+### Non Repudiation 
+Gebruikers kunnen een behoorlijk belang hebben bij een negatieve test uitslag omdat een postieve verplicht tot quarantaine. Indien er feedback 
```

Comment:
Hier mist volgens mij iets nog. Verder moet er misschien een bron of nuance bij "verplicht"

Reply:
Gaak ik aanpassen.